sass/utils/_variables.scss

```diff
@@ -3,6 +3,8 @@
 // Color Variables
 $primary-color: #166de0;
 $primary-color--hover: darken($primary-color, 10%);
+$delete-color: #d24b4e;
+$delete-color--hover: darken($delete-color, 10%);
```

Comment:
@AshishDhama i know you have raised this before in post of adding new css vars

Reply:
yeah `--dnd-indicator`

Reply:
yep, `--dnd-indicator` should be the right variable instead

Reply:
Updated to use `--dnd-indicator`